### PR TITLE
Setup dev stage envs

### PR DESCRIPTION
**Pull request by Minkov**

### CODE REVIEW

File: docker-compose.dev.yml [hunk 1]
```diff
@@ -2,23 +2,30 @@ version: '3.4'
 
 services:
   nginx:
-    build: .
+    image: minkov/nginx:latest
```
Comment (jicata) [inline docker-compose.dev.yml:5]: is this the real location of the image?

File: docker-compose.yml [hunk 2]
```diff
@@ -2,30 +2,30 @@ version: '3.4'
...
+      - ./local/conf.d:/etc/nginx/conf.d
+      - ./local/lua:/etc/nginx/lua
       - ./logs:/var/log/nginx
-      - ./shared:/etc/nginx/shared
+      - ./local/shared:/etc/nginx/shared
       - ./data/certbot/conf:/etc/letsencrypt
       - ./data/certbot/www:/var/www/certbot
     extra_hosts:
-      - "localnode:172.18.0.1"
+      - "localnode:192.168.14.138"
```
Comment (jicata) [inline docker-compose.yml]: What is this IP?

**Review by jicata — CHANGES_REQUESTED** [review]

**Comment by jicata** [comment]
Also two more points:

1. The payment products redirects seem to be missing here. You can look them up from the old config and reference them with some of the related issues like for example: https://github.com/SoftUni-Internal/interactive-issues/issues/754 and https://github.com/SoftUni-Internal/interactive-issues/issues/696
2. The default timeout request time must be increased in the nginx configuration at least for the administration servers due to our sometime long-running "temp" methods

File: docker-compose.dev.yml (hunk 1 above)
Comment (Minkov) [reply docker-compose.dev.yml:5]: Yes, for now

File: docker-compose.yml (hunk 2 above)
Comment (Minkov) [reply docker-compose.yml]: A local IP on my machine. Leave it as is

**Pull request merged**